README.md

```diff
@@ -89,6 +89,13 @@ pkgs
 [interactions]: #interactions
 
 - `nix edit` is unaffected, since it uses a packages `meta.position` to get the file to edit.
+- `git blame` locally and on GitHub is unaffected, since it follows file renames properly.
+- A commonly recommended way of building package directories in nixpkgs is to use `nix-build -E 'with import <nixpkgs> {}; callPackage pkgs/applications/misc/hello {}'`.
+  Since the path changes `pkg-fun.nix` is now used, this becomes like `nix-build -E 'with import <nixpkgs> {}; callPackage pkgs/unit/he/hello/pkg-fun.nix {}'`, which is harder for users.
+  However, calling a path like this is an anti-pattern anyways, because it doesn't use the correct nixpkgs version and it doesn't use the correct argument overrides.
```

Comment:
```suggestion
  However, calling a path like this is an anti-pattern anyway, because it doesn't use the correct nixpkgs version and it doesn't use the correct argument overrides.
```